docs/source/release/v5.1.0/mantidworkbench.rst

```diff
@@ -5,172 +5,272 @@ Workbench Changes
+- Instrument view now updates on zoom with a mouse wheel.
+- Failing to open for a certain workspace now outputs useful error messages.
+
+
+Misc
+####
+
+- Added a Clear button to the workspaces toolbox which removes all workspaces from the tree including hidden ones
+- The workbench launch scripts have been replaced by an executable on macOS & Windows. On Windows this will stop virus scanners
+  flagging the old ``launch_workbench.exe`` as a threat and quarantining it.
 - Improved the usability of the fit function and peak selection pop-up menus by allowing the user to immediately search for the desired function and activate autocompletion by pressing "enter" if there is just a single possible function.
-- The figure options menu now has a help button which opens the documentation for the menu.
-- Added a profiling option to the workbench launch script, allowing for timing of startup and other internal processes.
-- Variables assigned in python scripts are now cleared when a script is run in its entirety.
-- The colorbar on colorfill plots is now labelled, and the label can be set in the figure options.
-- Monitors are no longer shown on bin and colorfill plots.
-- User data directories are no longer checked at startup, reducing launch times with slow network drives.
-- When choosing a marker in the figure options, if one of the marker colours would not be used that selection is disabled.
-- There are now options in the settings window and the figure options for showing minor ticks and minor gridlines on plots.
-- Added an option to set the default ```drawstyle``` within the workbench settings window. Additionally, the ```linestyle``` can now be set to 'None'.
-- Added a button to the workbench settings window to save and load the settings to and from a file so they can be shared with others
+- Slow running algorithms will now display an estimated time to completion in their progress bars.
+- Algorithm Dialogs with a Combo box (Drop-down list) for workspace selection, are now editable to support searching and autocompletion.
 - Added an option to matrix workspaces to export bins and spectra to a table workspace.
-- Improved the handling of ``WorkspaceSingleValue`` workspaces in workbench. This fixes a crash which occurred when interacting with workspaces of this type.
-- Right-clicking a plot without dragging while using the zoom tool now resets the axes limits.
-- The Slice Viewer now starts with the zoom option selected by default.
-- The curves in the dropdown list in the Curves tab of the figure options are now listed in the same order as the plot legend.
+- ``TableWorkspaces`` can now have columns containing errors linked to corresponding columns containing values, using the ``setLinkedYCol(errColumn, dataColumn)``.
+- Use Jemalloc for memory allocation on Linux so memory can be released to the system.
 
-.. figure:: ../../images/instrument_view_sector.png
-   :align: right
-   :width: 400px
+Launch
+^^^^^^
```

Comment:
```suggestion
######
```

Reply:
Sure I'll change and move these sections above Misc!